clarity/src/vm/functions/arithmetic.rs

```diff
@@ -441,6 +482,64 @@ pub fn native_mod(a: Value, b: Value) -> InterpreterResult<Value> {
     type_force_binary_arithmetic!(modulo, a, b)
 }
 
+pub fn native_bitwise_left_shift(input: Value, pos: Value) -> InterpreterResult<Value> {
+    if let Value::UInt(u128_val) = pos {
+        let shamt = u32::try_from(u128_val & 0xffffffff)
```

Comment:
I almost wonder if this mask should really be `0x7f`, since there's only 128 bits anyway.

Reply:
That's what's happening under the hood any way in `wrapping_shl`, but maybe it would be more clear to a reader of this code of it was explicit here.